### PR TITLE
feat(JoJo's Bizarre Encyclopedia): add presence

### PR DESCRIPTION
**Pull request by wavpro**
## Description 
Added new presence for JoJo's Bizarre Encyclopedia (jojowiki.com and jojo-news.com). It displays activity on the mentioned sites.

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Screenshot

Screenshot

Screenshot

Screenshot

</details>

### CODE REVIEW

**Comment by wavpro** [comment]
It seems to fail checks because of other presences, not mine. Github says my fork was fully up to date, can someone check this out?

**Comment by EncryptedDev** [comment]
CI failed on your PR because you edited the presence. Delete the changes

**Comment by wavpro** [comment]
> CI failed on your PR because you edited the presence. Delete the changes

How do I remove it from the pull request? It doesn't help editing since nothing has changed from the old version, just that there's a commit that says it's changed. I don't know why this happened, I didn't see these files included when committing.

**Comment by RisingSunLight42** [comment]
> > CI failed on your PR because you edited the presence. Delete the changes
> 
> How do I remove it from the pull request? It doesn't help editing since nothing has changed from the old version, just that there's a commit that says it's changed. I don't know why this happened, I didn't see these files included when committing.

Those files are from your yarn format, those two files has a little format errors that make them change when you run the commande.
Rollback you commits to fix that

**Comment by wavpro** [comment]
I've fixed the formatting error

**Comment by Bas950** [comment]
Undo the packagejson and yarnlock too

**Comment by wavpro** [comment]
> Undo the packagejson and yarnlock too

Done

**Comment by EncryptedDev** [comment]
> > Undo the packagejson and yarnlock too
> 
> Done

Nope, `package.lock` and `yarn.lock` are still on this PR

**Comment by wavpro** [comment]
How would I remove them completely from the PR? I have made sure they're
exactly the same as currently in the repo.

File: websites/J/JoJo's Bizarre Encyclopedia/presence.ts [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+	clientId: "1007688410875564062",
+});
+
+presence.on("UpdateData", () => {
+	let presenceData: PresenceData = {};
+
+	if (window.location.href.toLowerCase().includes("jojowiki.com")) {
+		presenceData = {
+			largeImageKey: "logo_bordered",
+			smallImageKey: "wiki_icon",
```
Comment (Bas950) [inline websites/J/JoJo's Bizarre Encyclopedia/presence.ts:11]: please use imgur links

File: websites/J/JoJo's Bizarre Encyclopedia/presence.ts [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+						url: window.location.href,
+					},
+				];
+			}
+		}
+	}
+	if (window.location.hostname.toLowerCase().includes("jojo-news.com")) {
+		presenceData = {
+			largeImageKey: "logo_bordered",
+			smallImageKey: "news_icon_bordered",
```
Comment (Bas950) [inline websites/J/JoJo's Bizarre Encyclopedia/presence.ts:128]: same here

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**